plugins/vbe/fit_test.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Library for U-Boot Flat Image Tree (FIT)
+ *
+ * Copyright (C) 2022 Google LLC
+ * Written by Simon Glass <sjg@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#ifndef __FU_PLUGIN_VBE_FIT_TEST_H
+#define __FU_PLUGIN_VBE_FIT_TEST_H
+
+int
+fit_test(void);
+
+#endif /* __FU_PLUGIN_VBE_FIT_TEST_H */
```

Comment:
this whole file isn't needed AFAICT.  It's just used by the test suite right?  Can't you just make `fit_test` static to that source file?

Reply:
Yes not needed now we use g_test. Done